src/styles/_catalog.scss

```diff
@@ -4,6 +4,7 @@
 
   max-width: 944px;
   margin: 0 auto;
+  perspective: 100px;
```

Comment:
You don't need perspective here in case when you use "transform: scale"
It used with "translate"